client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import i18n from 'lib/mixins/i18n';
+import PurchaseDetail from './purchase-detail';
+
+const BusinessPlanDetails = ( { isFreeTrial, selectedSite } ) => {
+	var showGetFreeDomainTip = ! isFreeTrial;
```

Comment:
Well we already updated these components to make them stateless so I guess we could fix a few additional things in the process. I would not do that with the `CheckoutThankYou` component though since it's a much more bigger piece and we would need more time before deploying this pull request.